coalib/bearlib/aspects/Smell.py

```diff
@@ -324,3 +542,46 @@ class docs:
         'This the maximum number of embedded branches or embedded loops'
         ' allowed.',
         (6,), default=6)
+
+
+@Complexity.subaspect
+class MaintainabilityIndex:
+    """The `maintainablity index
```

Comment:
One line summary.  no URLs.

The details appear in the docstring after a blank line.